src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -10,7 +10,7 @@ import {
 } from '../../../types/database/schema/PostNotificationChannelList.types';
 import {MessageChannelItemProps} from '../../../types/component/AnonymousChat/MessageChannelItem.types';
 import {calculateTime} from '../../utils/time';
-import {capitalizeFirstText} from '../../utils/string/StringUtils';
+import {capitalizeFirstText, getOfficialAnonUsername} from '../../utils/string/StringUtils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `getOfficialAnonUsername` function is a good addition. It helps to maintain consistency in the way anonymous usernames are generated across different components.
```diff
- import {capitalizeFirstText} from '../../utils/string/StringUtils';
+ import {capitalizeFirstText, getOfficialAnonUsername} from '../../utils/string/StringUtils';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->